components/_settings/_config.scss

```diff
@@ -0,0 +1,4 @@
+:root {
+  --color-accent: var(--color-theme-accent, var(--color-blue-medium));
+  --color-divider: var(--color-gray-500);
```

Comment:
I see this declared here, but I wasn't sure how it gets into the components. When running storybook I don't have `--color-divider` defined.

Reply:
Weird, I do see it defined. If you want, I can look at it with you, just to make sure. Otherwise, we'll continue assuming my environment is correct, and that it's there. If it's not somehow, we'll notice it pretty quick.

Reply:
I noticed this was updated in a later PR which I used to generate a throw-away theme on my local. Can confirm Brian's comments here.